frontend/src/app/map/map.component.html

```diff
@@ -0,0 +1,3316 @@
+<p>
+  map works!
+</p>
+<button mat-button>Click me!</button>
```

Comment:
Nie dodawałbym fragmentów, które nic nie wnoszą, bo potem nie wiadomo czy to usunąć czy nie + zazwyczaj merge conflicty powstają.